src/flows/pipes/Data/index.ts

```diff
@@ -17,7 +17,10 @@ export default register => {
     },
     generateFlux: (pipe, create, _append) => {
       const {aggregateFunction, bucket, field, measurement, tags} = pipe
-
+      if (!bucket) {
+        create('')
```

Comment:
does this send off a network request query of ``''`?

Reply:
I believe so. This was more to guard against times when a bucket isn't selected, do you think we should just return in those cases?

Reply:
yeah try that out. there's also this example for making a query pipe a passthrough when there's no data: https://github.com/influxdata/ui/blob/master/src/flows/pipes/Query/index.ts#L35